src/mmw/js/src/core/utils.js

```diff
@@ -607,6 +609,34 @@ var utils = {
             min: min,
             max: max
         };
+    },
+
+    // Parses version number from branch and git describe output
+    // Outputs version number and release notes URL
+    parseVersion: function(branch, gitDescribe) {
+        var version = 'Unknown',
+            url = RELEASE_NOTES_BASE_URL + '/latest';
+
+        if (branch === null && gitDescribe === null) {
+            version = 'Unknown';
+        } else if (branch === 'local' && gitDescribe === null) {
+            version = 'Local';
+        } else if (branch.startsWith('release')) {
+            version = branch.substr(branch.indexOf('/') + 1);
+            // Use this version for release notes
+            url = RELEASE_NOTES_BASE_URL + '/tag/' + version;
+        } else if (branch.startsWith('hotfix')) {
+            version = branch.substr(branch.indexOf('/') + 1);
+            // Use the original release notes
+            url = RELEASE_NOTES_BASE_URL + '/tag/' + version.match(MINOR_MAJOR_REGEX) + '0';
+        } else {
+            version = gitDescribe.substr(0, 18);
```

Comment:
This is failing with the error below on staging. I believe that `gitDescribe` is coming through as `undefined`.

```
Uncaught ReferenceError: getDescribe is not defined
    at eval (eval at parseVersion (main.js:12468), <anonymous>:1:1)
    at Object.parseVersion (main.js:12468)
    at child.templateHelpers (main.js:9418)
    at Object.Marionette._getValue (vendor.js:45472)
    at child.mixinTemplateHelpers (vendor.js:46600)
    at child._renderTemplate (vendor.js:47004)
    at child.render (vendor.js:46975)
    at child.showAbout (main.js:12626)
    at HTMLElement.dispatch (vendor.js:71729)
    at HTMLElement.elemData.handle (vendor.js:71415)
```

Reply:
Interesting. The [`/version.txt`](https://staging.app.wikiwatershed.org/version.txt) on staging has items separated by a space instead of tab:

```
origin/tt/version-number 1.21.0-131-gec9d821c4a90edca8c2d62416e05975dd5ab7eb9
```

I'll adjust it to match any whitespace at all.

Reply:
I think it's working now:

![image](https://user-images.githubusercontent.com/1430060/34729157-e7bc89b0-f529-11e7-9f23-21305d859441.png)